### DISCUSSION TITLE
How to Stop VpnHood Server on Windows?

### DISCUSSION THREAD

**Discussion by momanan** [vpnhood/VpnHood#609] [General]
How to Stop VpnHood Server on Windows?
[Net.ServicePointManager]::SecurityProtocol = "Tls,Tls11,Tls12"; iwr("https://github.com/vpnhood/VpnHood/releases/latest/download/VpnHoodServer-win-x64.ps1")|iex;

**Comment by trudyhood** [comment]
Hi,
Read this:
https://github.com/vpnhood/VpnHood/wiki/How-to-uninstall-VpnHood-Server-from-Windows

**Reply by momanan** [reply]
I'm thankful
